mmaction/models/backbones/resnet3d.py

```diff
@@ -999,5 +1000,5 @@ def train(self, mode=True):
                     m.eval()
 
 
-if 'mmdet' in dir():
+if mmdet_imported:
     MMDET_SHARED_HEADS.register_module()(ResNet3dLayer)
```

Comment:
This line is too weird. The following is more straight forward.

```python
MMDET_SHARED_HEADS.register_module(ResNet3dLayer)
```

Reply:
However, registry.register_module() return a function, 
```python
@MMDET_SHARED_HEADS.register_module()
class ResNet3dLayer():
    pass
```
is equivalent to: 
```python
MMDET_SHARED_HEADS.register_module()(ResNet3dLayer)
```